python/amici/sbml_import.py

```diff
@@ -1625,16 +1625,16 @@ def _parse_trigger(trigger: sp.Expr) -> sp.Expr:
         # translating into aheaviside function that is implemented as unit step
         if isinstance(trigger, sp.core.relational.StrictLessThan):
             # x < y => x - y < 0 => r < 0
-            return sp.Heaviside(-root)
+            return 1 - sp.Heaviside(root)
         if isinstance(trigger, sp.core.relational.LessThan):
             # x <= y => not(y < x) => not(y - x < 0) => not -r < 0
-            return 1 - sp.Heaviside(root)
+            return sp.Heaviside(-root)
```

Comment:
What's the difference between `1-H(x)` and `H(-x)`? Should the difference between `< ` and `<=` be the value that `H(x)` is defined to take at `x==0`? Similarly for `>` and `>=` below?

Reply:
Ah, nevermind https://github.com/AMICI-dev/AMICI/blob/a5c679b0cce90e192bacf9461d43825de8f675ce/src/symbolic_functions.cpp#L90-L95

Reply:
Hm, given this definition, `x < 0` would correspond to `H(-x)`?

Reply:
Actually, `1-H(x)` and `H(-x)` should be equivalent... For the difference between `<` and `<=` I have to say: Hard to tell what will happen if x is a floating point number...

Reply:
> `1-H(x)` and `H(-x)` should be equivalent

Not equivalent for this implementation of the Heaviside function but agreed :+1: (equivalent everywhere except 0)

Reply:
This implementation addresses the situation when the simulation starts at a point where some of the arguments of heaviside functions evaluates to 0 (which can happen when translating SBML piecewise functions), which led to errors previously. It looks like the cpp implementation needs to be updated as its not consistent with the internal implementation. Will add a short blurb about that to the documentation.